docs/components/modeler/web-modeler/save-as-element-templates.md

```diff
@@ -0,0 +1,161 @@
+   prompting you to ensure that these dependencies are available in the runtime context where the template will be used.
+   The dependencies themselves are not included in the template and must be managed and deployed separately.
+
+   <img src={PublishTemplateImg6} alt="Publish template dialog with version fields, description text area, and a warning about decision dependency for the determine-fraud-rating decision" style={{marginTop: '0', width: '800px'}} />
+
+### Step 4: Using your saved templates
+
+Once you've created and published a template, you can use it in a diagram:
+
+1. Add a new task to your diagram or select an existing task.
+2. You can apply a template by clicking on a task and selecting the **Change element** menu icon. Alternatively, in the properties panel, click the **Select** button in the Template section.
+3. In the template selection dialog, you'll find your template (for example, "Determine fraud rating confidence template") listed
+   under the category that you assigned. By default, new templates are in the category "Templates".
+4. Select the template to apply it to your task. The task will be automatically configured with all the settings from your template, including decision references, bindings, and variable mappings.
+   <img src={ApplyTemplateImg7} alt="Template selection sidebar showing available connectors and templates, with 'Determine fraud rating confidence template' listed under the Templates section" style={{marginTop: '0', width: '800px'}} />
+
+## Understanding template property bindings
+
+When you save a task as a template, Web Modeler automatically converts the task's properties into appropriate template bindings:
+
+- Input/output mappings
+- Task headers
+- Zeebe properties
+- Element-specific properties (for example, calledDecision, calledElement)
+- Message references (for message-related tasks)
+
+All properties are configured with the appropriate binding types to ensure the template works correctly when applied to a task.
+Only properties supported by element templates are included in the template.
+
+Unsupported properties are ignored and remain visible in the properties panel after you apply the template.
+For a list of supported properties, refer to the [Element templates reference](/components/modeler/desktop-modeler/element-templates/defining-templates.md#defining-template-properties).
+
+## When template creation is unavailable
+
+The "Save as template" button is disabled in the following scenarios:
+
+- You are not in Web Modeler's implementation mode
+- The task has active validation issues
+- The element type is not supported (Blank Task, Events, Subprocesses, etc.)
+- You don't have the necessary permissions to create templates
+
+Resolving validation issues will enable the "Save as template" button if the task type is supported.
+
+## Best practices
+
+When creating templates from tasks, follow these best practices:
```

Comment:
[🔧] Should we add something around versioning names ? Maybe suggest trying to use semver